test/dash/dash_parser_manifest_unit.js

```diff
@@ -1370,4 +1370,36 @@ describe('DashParser Manifest', () => {
     expect(textStream.roles).toEqual(['captions', 'foo']);
     expect(textStream.kind).toBe('caption');
   });
+
+  // Filter image adaptation sets, until we add thumbnail track support.
+  // See: https://github.com/google/shaka-player/issues/559
+  it('Filter image adaptation sets', async () => {
```

Comment:
Nit: This is not an accurate name for this test. It does nothing actually check if the image track is filtered out; it only tests to see if the manifest is parsed without an error being thrown. Once we add thumbnail track support, this test will still pass, even though we will no longer be filtering image adaptation sets then.

A better name would be:
```js
  it('does not error when image adaptation sets are present', async () => {
```

Reply:
I have changed it